src/test/java/com/qa/restassured/VerifyNameWithNonStaticImports.java

```diff
@@ -0,0 +1,18 @@
+package com.qa.restassured;
+
+import io.restassured.RestAssured;
+
+import org.hamcrest.Matchers;
+
+public class VerifyNameWithNonStaticImports { //Program for Non-Static Imports
+
```

Comment:
I think the @Test annotation is missing here

Reply:
Modify